modules/distr-tracing-rn-new-features.adoc

```diff
@@ -11,11 +11,13 @@ Result – If changed, describe the current user experience.
 [id="distr-tracing-rn-new-features_{context}"]
 == New features and enhancements {ProductName} 2.0.0
 
-This release marks the rebranding of Red Hat OpenShift Jaeger to {ProductName}.  This effort includes the following:
+This release marks the rebranding of Red Hat OpenShift Jaeger to {ProductName}. This name change is to reflect that the distributed tracing solution encompasses more than just Jaeger project components, but also Elasticsearch, AMQ Streams, and OpenTelemetry. The rebranding also allows for new technologies and components to be added in the future under the distributed tracing name.
 
-* Updates {ProductShortName} Operator to Jaeger 1.28.  Going forward, {ProductName} will only support the `stable` Operator channel. Channels for individual releases are no longer supported.
+This release includes the following:
 
-* Introduces a new OpenTelemetry Operator based on OpenTelemetry 0.33.  Note that this Operator is a Technology Preview.
+* Updates {ProductShortName} Operator to Jaeger 1.28. Going forward, {ProductName} will only support the `stable` Operator channel. Channels for individual releases are no longer supported.
+
+* Introduces a new OpenTelemetry Operator based on OpenTelemetry 0.33. Note that this Operator is a Technology Preview.
```

Comment:
We shouldn't refer to this as the 'OpenTelemetry Operator', we should refer to this as the 'Red Hat OpenShift distributed tracing data collection' operator which is based on OpenTelemetry 0.33